api.txt

```diff
@@ -79,3 +79,26 @@ The followed up with a move user_moved event
 When a user creates a portal
 
 {"method":"user_portal", "data":{"userId":"LL","roomId":"345678354764987457","url":"http://...", "pos":[0,0,0], "fwd":[0,1,0]}}
+
+
+Request how many users are connected to the server:
+{"method":"getusersonline"}
+
+Returns:
+{"data": 2, "method": "usersonline"}
+
+To get user count in the current room (NOT WORKING YET, WILL RETURN 0 FOR THE TIME BEING)
```

Comment:
Personally, I typically prefer to not commit non-working code to a project, but I'll leave that up to @lisa-lionheart